kola/tests/ignition/passwd.go

```diff
@@ -147,18 +147,94 @@ func init() {
 		               ]
 		             }
 		           }`),
-		Distros: []string{"cl", "rhcos"},
+		Distros: []string{"cl"},
+	})
+	register.Register(&register.Test{
+		Name:        "rhcos.ignition.v1.users",
+		Run:         usersRhcos,
+		ClusterSize: 1,
+		UserData: conf.Ignition(`{
+		             "ignitionVersion": 1,
+		             "systemd": {
+		               "units": [{
+		                 "name": "system-cloudinit@usr-share-coreos-developer_data.service",
```

Comment:
Does this service actually exist on RHCOS machines?

Reply:
No it doesn't exist in RHCOS.

I didn't find it in CL either:
`[bound] core@ip-172-31-21-214 ~ $ systemctl list-units | grep system-cloudinit`

Should it be dropped from both RHCOS and CL?  I wasn't sure if it was a drop in for something I wasn't aware of so I left it in.

Reply:
Yes, developer images are gone, but that password can still be set in prod images with `set_shared_user_password.sh`.  (It can actually take any cloud-config file, but that password script is the normal use for it.)

Reply:
I'll remove it for RHCOS and leave it as is for CL.